donut.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+(function() {
+  var DonutShop = function (minCus, maxCus, average, store, open, close) {
+    this.minCus = minCus;
+    this.maxCus = maxCus;
+    this.average = average;
+    this.store = store;
+    this.open = open || 700;
+    this.close = close || 1800;
+    this.hoursOpen = (this.close - this.open)/100;
+    this.hourTotal = [];
+  };
+
+  DonutShop.prototype.donutperhour = function () {
+    console.log("donuthour");
```

Comment:
It's okay to work with console.logs in your debugging and builds to test. When you submit work in a professional setting, keep in mind that you'll want to remove them from your code before commit/push.